build/templates/session.py.mako

```diff
@@ -218,16 +236,30 @@ constructor_params = helper.filter_parameters(init_function['parameters'], helpe
 class Session(_SessionBase):
     '''${config['session_class_description']}'''
 
-    def __init__(${init_method_params}):
+<% grpc_channel_param = ", *, grpc_channel=None" if grpc_supported else "" %>\
```

Comment:
I had to look up what this `*` argument was (means that any arguments after it can only be passed as keyword arguments). Might be worth adding a comment (at least in the template, if nowhere else).

Reply:
I disagree about adding a comment. It's a standard python thing now, and should be a learn-it-once sort of thing for python devs (across all python projects that they work on).